manning/manning-proposal-2020-07.md

```diff
@@ -77,7 +75,7 @@ Filling in these gaps will help researchers get more done in less time and with
 6.  Adopt a license, a code of conduct, and efficient decision-making processes to ensure the project runs smoothly.
 7.  Publish results with DOIs, ORCIDs, and other metadata in a fully-reproducible way.
```

Comment:
Point 7 on publishing might need to be taken out if we adopt the proposal being discussed in #415.

Reply:
It could be replaced with: "7. Document, package and release software for use by the wider research community"

Reply:
:+1: